dependencies.yaml

```diff
@@ -1,81 +0,0 @@
-dependencies:
```

Comment:
as this is removed to simplify dependencies, what would be a new way of installing a wanted set of versions of services operators?

Reply:
If you want to use a custom image it depends on your workflow. If you build images with the github actions workflow and post them to quay.io automatically based on your custom branches it is simply a matter of 1) updating the go.mod, and 2) running 'make bindata'. If you just want to splice in your own manually built operator you can update the images in config/operator/manager_operator_images.yaml directly.

Reply:
I was also thinking for testing minor updates by "human operators" or developers acting like users, when the source versions of operators should be picked from earlier releases